localization/l10n/bundle.l10n.json

```diff
@@ -305,6 +305,87 @@
   "Search...": "Search...",
   "Maximize": "Maximize",
   "Restore": "Restore",
+  "To compare two schemas, first select a source schema and target schema, then press compare.": "To compare two schemas, first select a source schema and target schema, then press compare.",
+  "No schema differences were found.": "No schema differences were found.",
+  "Initializing comparison, this might take a while...": "Initializing comparison, this might take a while...",
+  "default": "default",
```

Comment:
Is this right casing?

Reply:
Yeah, that's the default casing. I verified with Azure Data Studio too and that can be found here if you'd like to take a look too: https://github.com/microsoft/azuredatastudio/blob/d6fe81f3c10ea1029eb9f781261f7b7b3d149a9e/extensions/sql-database-projects/src/common/constants.ts#L149